scripts/download_install_deformable.sh

```diff
@@ -0,0 +1,8 @@
+mkdir xrmocap_download
+cd xrmocap_download
+wget -q https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrmocap/Deformable.tar
```

Comment:
After checking out this tar, I think we may maintain the cuda code directly in our codebase, which would be easier to track if anything from Deformable is updated.

We can leave it for another PR.